lib/rspec/core/example.rb

```diff
@@ -231,8 +231,13 @@ def example_group
         @example_group_class
       end
 
-      alias_method :pending?, :pending
-      alias_method :skipped?, :skip
+      def pending?
+        !!pending
+      end
+
+      def skipped?
+        !!skip
+      end
```

Comment:
These are actually superfluous, as they are always booleans due to the way it is assigned to metadata (they get overwritten to true)

Reply:
(From conversation in main thread, only the `skipped?` definition should be necessary.)

Reply:
Hmmm , indeed. Curious. I must have confused it with `pending_skipped`. Reverted that part. 👍